app/views/prior_authority/applications/_drafts.html.erb

```diff
@@ -40,8 +40,16 @@
             <%= application.laa_reference %>
           </td>
           <td class="govuk-table__cell">
-            <%= link_to t("prior_authority.applications.tabs.delete"),
-                        confirm_delete_prior_authority_application_path(application) %>
+            <a
```

Comment:
It doesn't really make a difference, but FYI you could have continued to use `link_to` rather than manually constructing the <a> html by doing:

```erb
<%= link_to confirm_delete_prior_authority_application_path(application) , class: "govuk-link--no-visited-state" do %>
  <%= t("prior_authority.applications.tabs.delete") %>
  <span class="govuk-visually-hidden">
    applications
    <%= application.laa_reference %>
  </span>
<% end %>
```

```

Reply:
Ah I tried something like that but was getting errors, thanks